currency_tests.py

```diff
@@ -0,0 +1,85 @@
+import currency
+import unittest
+from datetime import datetime
+
+
+class CurrencyTest(unittest.TestCase):
+
+    def test_date_check(self):
+        result = currency.date_check(str(datetime.today().date()))
+        self.assertEqual(result, True)
+
+    def test_less_date_check(self):
+        result = currency.date_check("1992-06-20")
+        self.assertEqual(result, False)
+
+    def test_higher_date_check(self):
+        date = str(datetime.today().year) + '-' + str(datetime.today().month) + '-' + str(datetime.today().day + 1)
+        result = currency.date_check(date)
+        self.assertEqual(result, False)
+
+    def test_13month(self):
+        result = currency.date_check("2019-13-10")
+        self.assertEqual(result, False)
+
+    def test_parse(self):
+        html = open("F:/project/start/xml.txt")

```

Comment:
This won't work on other machines. You have to use the path of your `currency_test.py` script, like this:

```python
import os

xml_path = os.path.join(
    os.path.dirname(os.path.realpath(__file__)),
    "xml.txt"
)

class CurrencyTest(unittest.TestCase):
    # ...
    def test_parse(self):
        html = open(xml_path)
        # ...
```

Also, there is a very useful operator in python which allows you to handle file closing in a hands-free way:
```python
def test_parse(self):
    # Once the `with` operator goes out of scope, it runs `f.close()` automatically
    with open(xml_path) as f:
        html = f.read()

    self.assertEqual(currency.parse(html, "usd", False), "30.9436")
```